service/src/main/java/bio/terra/workspace/service/resource/model/WsmResourceType.java

```diff
@@ -113,6 +115,13 @@ public enum WsmResourceType {
       ApiResourceType.GIT_REPO,
       ReferencedGitRepoResource.class,
       ReferencedGitRepoHandler::getHandler),
+  REFERENCED_TERRA_WORKSPACE(
+      CloudPlatform.ANY,
+      StewardshipType.REFERENCED,
+      "REFERENCED_TERRA_WORKSPACE",
```

Comment:
To follow the naming convention, this should be `REFERENCED_ANY_TERRA_WORKSPACE`

Reply:
Done